src/target/riscv/riscv-013.c

```diff
@@ -1726,6 +1777,12 @@ static int examine(struct target *target)
 		dmi_write(target, DM_DMCONTROL, DM_DMCONTROL_DMACTIVE);
 		dm->was_reset = true;
 	}
+
+	if (dbgbase_verify(target) == false) {
```

Comment:
You can use `(!expr)` instead of `(expr == false)`:

```suggestion
	if (!dbgbase_verify(target)) {
```